# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/canvas/WebGLTypes.h
# Commit: 6e9a89ead3a5
# Full Hash: 6e9a89ead3a52864c48a4e5481e9a35e4aad5e98
# Author: Jeff Gilbert <jgilbert@mozilla.com>
# Date: 2020-06-11 04:04:33
# Regressor Bug: 1632249
# File Overlap Count: 1
# Description:
#   Bug 1632249 - WebXR compositing fixes. r=imanol
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D78117
# ==============================================================================

diff -r dd00e2da3a0f -r 6e9a89ead3a5 dom/canvas/WebGLTypes.h
--- a/dom/canvas/WebGLTypes.h	Wed Jun 10 22:21:02 2020 +0000
+++ b/dom/canvas/WebGLTypes.h	Wed Jun 10 22:21:04 2020 +0000
@@ -990,26 +990,6 @@
 }
 
 // -
-// ClientWebGLContext exports!
-// (So that you don't need to include the mess that is ClientWebGLContext.h)
-namespace gfx {
-class SourceSurface;
-}  // namespace gfx
-
-namespace layers {
-class KnowsCompositor;
-class SurfaceDescriptor;
-}  // namespace layers
-
-namespace webgl {
-
-void Present(ClientWebGLContext&);
-Maybe<layers::SurfaceDescriptor> GetFrontBuffer(ClientWebGLContext&,
-                                                layers::KnowsCompositor*);
-
-}  // namespace webgl
-
-// -
 
 }  // namespace mozilla
 